README.md

```diff
@@ -0,0 +1,141 @@
+2. 📄 [Inception of rotating roles](https://medium.com/@laura.barcziova/inception-of-rotating-roles-9caf971b3096)
+3. 📄 [Share team responsibilities efficiently](https://medium.com/@laura.barcziova/share-team-responsibilities-efficiently-9a202aad7bd0)
+4. 📄 [Role rotation tutorial](https://medium.com/@laura.barcziova/role-rotation-tutorial-957ed3545ef2)
+5. 📽️ [A journey of iteration and role rotation for better collaboration - DevConf.CZ 2024](https://www.youtube.com/watch?v=y1t7Wd31bL8)
+
+
+## Common team roles
+We have interviewed several colleagues of ours from Red Hat. The conclusion we
+reached was there are multiple common roles that apply in many teams. We will
+define them here so you can easily reuse the roles in your team. They don’t
+apply everywhere and to everyone —  you are welcome to be inspired and create a
+role that will fit your team.
+
+## 🗓️Meeting lead
+Having a productive meeting that sticks to agenda, has notes, and a clear
+outcome, has a tremendeous impact on everyone. With one responsible meeting
+lead, you can stop wasting time of all people on your meetings.
+
+### Responsible for
+* Run a meeting (TBD: this could be more detailed)
+* Make sure people stay on topic
+* Follow meeting agenda - address all points
+* Write down action items and assign them
+* Make meeting notes or assign someone on the call to write them down
+
+### Why have this role?
+* Meetings need moderators, otherwise they succumb to chaos
+* Well run meeting is the difference between a complete waste of time and
+  effectively coordinating in a group of people
+
+### What can people learn?
+* Not everyone likes the attention, so this can be a big step outside of
+  comfort zone for many.
+* It can be tiring to be in charge of everything all the time. Managers
+  understand. Team members may suddenly feel more empathy and compassion once
+  they realize that leading a meeting requires significant effort.
+* Better knowledge about the code base to be able to review the changes.
+
+## 🏃 Agile lead
+As we participate in our agile methodology technique, be it scrum or kanban, we
+should have someone to be in charge. This role can be very similar to the
+Meeting lead. We are keeping it separate here so we can clearly distinguish
+between agile development methodology and just running meetings.
+
+### Responsible for
+* The team sticks to all the agile ceremonies as they are defined.
+* Coordinate with the meeting lead are be one.
+* Make sure everyone has the capacity to work on their tasks.
+* Assist people who are struggling in their assignments.
+* Volunteer if possible for i.e. research, testing, or reaching out to
+different teams if needed.
+* Celebrate success with the team.
+
+### Why have this role?
+* It could become chaotic without an agile lead.
+
+### What can people learn?
+* The responsibility to be in charge of the team's sprint.
```

Comment:
I would prefer not explicitly writing `sprint`, but not sure how to generalise it :thinking:

Reply:
yep, that's exactly what I was struggling with

Reply:
maybe something like iteration? or team's work for dedicated time unit :D but then `sprint` might be better in the end

Reply:
I like iteration, that's a good suggestion.